x-pack/plugins/spaces/public/management/spaces_management_app.tsx

```diff
@@ -73,6 +74,10 @@ export const spacesManagementApp = Object.freeze({
           import('./edit_space'),
         ]);
 
+        const { security } = await coreStart.plugins.onStart<{ security: SecurityPluginStart }>(
+          'security'
+        );
+
```

Comment:
I think we should stick to the same pattern we've adopted to inject utils exposed by the security plugin, through the space plugin entry [x-pack/plugins/spaces/public/plugin.tsx](https://github.com/elastic/kibana/blob/main/x-pack/plugins/spaces/public/plugin.tsx#L93-L115), wdyt?

Reply:
Good suggestion, it is much cleaner now.